cobrautil.go

```diff
@@ -32,6 +49,8 @@ func SyncViperPreRunE(prefix string) CobraRunFunc {
 			return nil // No-op for builtins
 		}
 
+		ReadDotEnv(prefix)
```

Comment:
Rather than modifying this function, let's add a new one:

```go
func SyncViperDotEnvPreRunE(prefix string) CobraRunFunc {
	return func(cmd *cobra.Command, args []string) error {
		// Maybe add local here if it's standard
		godotenv.Load()
		return SyncViperPreRunE(prefix)
	}
}
```

Reply:
renamed